h/static/scripts/directive/via-dialog.coffee

```diff
@@ -0,0 +1,27 @@
+###*
+# @ngdoc directive
+# @name viaLinkDialog
+# @restrict A
+# @description The dialog that generates a via link to the page h is currently
+# loaded on.
+###
+module.exports = ['$timeout', 'crossframe', ($timeout, crossframe) ->
+    viaUrl = 'https://via.hypothes.is/h/'
```

Comment:
It is possible to add embed this uri in the template and read it from there. 
The question is whether we want to do it this way or not?